deployment/k8s/charts/components/migrationConsole/migration_services.yaml

```diff
@@ -0,0 +1,32 @@
+source_cluster:
```

Comment:
can you file a jira bug to fix the creation of this file.  I was starting a fix but got hung up on three other things.  The last issue that I was looking at was the config_watcher.py file wasn't normalizing the casing of the configmap key names into the jinja template properly.

Reply:
Raised: https://opensearch.atlassian.net/browse/MIGRATIONS-2383